provision/provision.sh

```diff
@@ -761,9 +761,13 @@ mysql_setup() {
     cp -f  "/srv/config/mysql-config/root-my.cnf" "/home/vagrant/.my.cnf"
     chmod 0644 "/home/vagrant/.my.cnf"
     echo " * Copied /srv/config/mysql-config/root-my.cnf          to /home/vagrant/.my.cnf"
-    
-    echo " * Setting the default database password for the root user"
-    mysqladmin -u root password root
+
+    if [[ "ERROR 1045 (28000): Access denied for user 'root'@'localhost' (using password: YES)" != "$(mysql -u root -proot)" ]]; then
```

Comment:
It's enough to check the return code of the command

Reply:
Well in this case can detect this error, if there are other errors with the return code we cannot debug easily.

Reply:
If there's an error other than the access denied it could cause issues that are difficult to debug, lets check the return code instead  and print any errors out